### PR TITLE
ENH: add context menu item to view guidance

### PR DESCRIPTION
**Pull request by nstelter-slac**
[Screencast from 07-18-2023 02:09:05 PM.webm](https://github.com/slaclab/slac-alarm-manager/assets/137955700/982afb88-a768-44f7-b68c-dd0be03494f6)

### CODE REVIEW

**Comment by nstelter-slac** [comment]
Added test

File: slam/alarm_tree_view.py [hunk 1]
```diff
@@ -113,6 +115,37 @@ def tree_menu(self, pos: QPoint) -> None:
                     self.context_menu.addAction(self.unacknowledge_action)
             self.context_menu.addAction(self.enable_action)
             self.context_menu.addAction(self.disable_action)
+            self.context_menu.addMenu(self.guidance_menu)
+    
+            # Make the entires from the config-page appear when alarm in tree is right-clicked
+            self.guidance_objects.clear()
```
Comment (jbellister-slac) [inline slam/alarm_tree_view.py:121]:
```suggestion
```
Duplicate line that can be removed, cleared again just a few lines below

File: slam/tests/test_alarm_configuration_widget.py [hunk 2]
```diff
@@ -56,3 +56,6 @@ def test_save_configuration(qtbot, alarm_item, mock_kafka_producer, enabled, lat
     assert values_sent['displays'][0]['details'] == ''
     assert values_sent['commands'][0]['title'] == 'How to run display'
     assert values_sent['commands'][0]['details'] == 'bash run_display.sh'
+
+    # Verify right click menu displays the guidance data
+
```
Comment (jbellister-slac) [inline slam/tests/test_alarm_configuration_widget.py:61]:
```suggestion
```
Can be removed, test was added to `test_alarm_tree_view`

**Review by jbellister-slac — APPROVED** [review]
Looks good to me!

**Pull request merged**